### DISCUSSION TITLE
Pre doesn’t respect dark theme on iOS

### DISCUSSION THREAD

**Discussion by mrexodia** [tbolt/boltcss#12] [General]
![CA5398CB-726F-4500-AD10-CA473061DDEC](https://user-images.githubusercontent.com/2458265/198855651-39df9880-2a19-40a1-9bc3-ba3176ba4a21.png)

Not sure if this is intentional (or already fixed, I saw something in #4 related to it), but reporting in case it’s unintended…

Thanks for your library, definitely something I’ll want to use!

**Comment by tbolt** [comment]
This is intentional to match `<code>` but im on the fence about it. Moved this to a discussion in case others want to chime in. I will play around with having `<pre>` use a dark background though.

**Reply by bueltge** [reply]
The problem is also still existent on Android devices. But not possible to simulate with the Chrome Dev Tools, stupid. Maybe you should remove the background and color statements.

https://github.com/tbolt/boltcss/blob/main/bolt.css#L243

```css
  background-color: whitesmoke;
  color: black;
```

**Comment by tbolt** [comment]
I updated `<pre>` and `<code>` to use the default background color used by many other elements. Looks like this now-
<img width="422" alt="Screen Shot 2022-11-07" src="https://user-images.githubusercontent.com/2279260/200457153-3a55a8d4-65bc-4132-a75f-d2da121c3b59.png">
